modules/top-level/output.nix

```diff
@@ -98,13 +98,26 @@ in
         in
         lib.unique (builtins.concatMap pluginWithItsDeps normalizedPlugins);
 
+      # Remove dependencies from all plugins in a list
+      removeDependencies = ps: map (p: p // { plugin = removeAttrs p.plugin [ "dependencies" ]; }) ps;
+
       # Separated start and opt plugins
       partitionedPlugins = builtins.partition (p: p.optional) allPlugins;
```

Comment:
Maybe let's name this `partitionedOptionalPlugins`, to help with the other partition